.github/file-filters.yml

```diff
@@ -1,3 +1,17 @@
-backend:
-  - '**/*.py'
-  - 'requirements*.txt'
+# This is used by the action https://github.com/dorny/paths-filter (which we have forked to https://github.com/getsentry/paths-filter)
+
+# TODO: There are some meta files that we potentially could ignore for both front/backend,
+# as well as some configuration files that should trigger both
+
+# backend == !frontend
```

Comment:
I would elaborate on here saying we do this because frontend is more well-defined (give the example that backend `**/*.py` and `requirements*.txt` isn't sufficient due to django templating).